source/v4/channels.yaml

```diff
@@ -106,6 +106,47 @@
         '404':
           $ref: '#/responses/NotFound'
 
+  '/teams/{team_id}/channels':
+    get:
+      tags:
+        - teams
+      summary: Get team public channels
+      description: |
+        Get a page team public channel list based on query string parameters - team id, page and per page.
```

Comment:
Minor grammatical change, `Get a page of public channels on a team based on query string parameters - team_id, page and per_page.`